src/app/components/NetworkRibbon/component/TutorialScreen.tsx

```diff
@@ -207,7 +256,7 @@ export function TutorialScreen(props: Props) {
       </div>
     </>
   );
-}
+};
 
 const Details = styled.div`
```

Comment:
I see this component contains plenty of styled components. I know you didn't add them but if we don't clean it up when we are dealing with legacy files, no one will :D Majority of them have standard property values so you can just use tailwind classes.

Reply:
:face_exhaling: I didn't even scroll that far down, yeah you're right.